### PR TITLE
Add docs on the recreate procedure

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

File: modules/ROOT/pages/clustering/databases.adoc [hunk 1]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
 
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate databases using a procedure
+
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`] procedure, which allows you:
+
+* To restore a database store to a specified backup, while keeping all the users and roles (metadata) for the database.
+
+* To make a database writable again after it has been lost due to a disaster.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:236]:
```suggestion
* To restore a database to a write mode again after it has been lost due to a disaster.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1819.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering/databases)
- [Clustering](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering)

File: modules/ROOT/pages/clustering/databases.adoc [hunk 2]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
 
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate databases using a procedure
+
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`] procedure, which allows you:
+
+* To restore a database store to a specified backup, while keeping all the users and roles (metadata) for the database.
+
+* To restore a database to a write mode again after it has been lost due to a disaster.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:236]:
```suggestion
* To restore a database to a write mode again after it has been lost (for example, due to a disaster).
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 3]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+
+The recreate procedure selects the most up-to-date seed from the defined seeding servers.
+If only a subset of the previous cluster members is specified as seeders, there is a risk of data loss.
+To avoid this, ensure you back up the current store before replacing it with a seed from a URI, so the latest state can be restored if needed.
+====
+
+The database in question can be in an `online` or `offline` state when it is recreated, but a successful operation starts the database regardless of the previous state.
+
+If CDC is enabled for the database, it will still be enabled after recreating the database, however the chain will be broken.
+See link:https://neo4j.com/docs/cdc/current/[CDC documentation] for more details. - general link ?? Do we have any more specific guidelines on the topic?
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:254]:
```suggestion
See link:https://neo4j.com/docs/cdc/current/[CDC documentation] for more details.
```

Comment (NataliaIvakina) [reply modules/ROOT/pages/clustering/databases.adoc:254]: This is a general link to the docs. Do we have any more specific guidelines on the topic?

File: modules/ROOT/pages/clustering/databases.adoc [hunk 4]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+
+Before recreating a database, any eventual quarantined states need to be addressed.
+?? For more information, refer to the Quarantined State documentation. ??
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:257]:
```suggestion
For more information, refer to the quarantined state documentation.
```

Comment (NataliaIvakina) [reply modules/ROOT/pages/clustering/databases.adoc:257]: I didn't find where to link this paragraph.

File: modules/ROOT/pages/clustering/databases.adoc [hunk 5]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+
+The recreate procedure selects the most up-to-date seed from the defined seeding servers.
+If only a subset of the previous cluster members is specified as seeders, there is a risk of data loss.
+To avoid this, ensure you back up the current store before replacing it with a seed from a URI, so the latest state can be restored if needed.
+====
+
+The database in question can be in an `online` or `offline` state when it is recreated, but a successful operation starts the database regardless of the previous state.
+
+If CDC is enabled for the database, it will still be enabled after recreating the database, however the chain will be broken.
+See link:https://neo4j.com/docs/cdc/current/[CDC documentation] for more details.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:254]: This is a general link to the docs. Do we have any more specific guidelines on the topic?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1819.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering/databases)
- [Clustering](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1819.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering/databases)
- [Clustering](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering)

File: modules/ROOT/pages/clustering/databases.adoc [hunk 6]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+
+Before recreating a database, any eventual quarantined states need to be addressed.
+For more information, refer to the quarantined state documentation.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:257]: I didn't find where to link this place.

File: modules/ROOT/pages/clustering/databases.adoc [hunk 7]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+
+You need xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-database-management[the DBMS `DATABASE MANAGEMENT` privileges] to run the recreate procedure. 
+
+[[recreate-seeding-options]]
+=== Seeding options and configurations
+
+The store to be used during the recreation can be defined in three different ways.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:264]:
```suggestion
The database store to be used during the recreation can be defined in three different ways.
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 8]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+
+[source, shell]
+----
+CALL dbms.cluster.recreateDatabase("neo4j", {seedingServers: ["serverId1", "serverId2", "serverId3"]});
+----
+
+[[undefined-servers]]
+==== Undefined servers
+
+If you provide an empty list of seeding servers or do not specify a `seedURI`, Neo4j automatically selects all available allocations of the database as seeders.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:297]:
```suggestion
If you provide an empty list of seeding servers and do not specify a `seedURI`, Neo4j automatically selects all available allocations of the database as seeders.
```

Comment (gfx54b) [reply modules/ROOT/pages/clustering/databases.adoc:297]: and

Either `seedURI` or `seedingServers` must be in the options. If you define neither that is an error.
If you define both then:
* either `seedingServers` is an empty list then see "Undefined servers with fallback backup"
* or `seedingServers` is not empty, that is an error

File: modules/ROOT/pages/clustering/databases.adoc [hunk 9]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate databases using a procedure
+
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`] procedure, which allows you:
+
+* To restore a database store to a specified backup, while keeping all the users and roles (metadata) for the database.
+
+* To restore a database to a write mode again after it has been lost (for example, due to a disaster).
+See xref:clustering/disaster-recovery.adoc[] for more information.
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/databases.adoc:237]: I think we should hold off on pointing to the disaster recovery docs here, until we have made the changes there.

File: modules/ROOT/pages/clustering/databases.adoc [hunk 10]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
 +----------------------------------------------------------------------------------------------------------------------------------------+
 ----
 
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate databases using a procedure
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/databases.adoc:230]: I don't think we need to specify this is a procedure here, since this is not done for Deallocate databases or Reallocate databases even though both of those are procedures as well.

File: modules/ROOT/pages/clustering/databases.adoc [hunk 11]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`] procedure, which allows you:
+
+* To restore a database store to a specified backup, while keeping all the users and roles (metadata) for the database.
+
+* To restore a database to a write mode again after it has been lost (for example, due to a disaster).
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/databases.adoc:236]: On both this and the previous one we have the word "restore". This is unfortunately very overloaded, could we find a nice way of saying it without this word?

File: modules/ROOT/pages/clustering/databases.adoc (hunk 5 above)
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/databases.adoc:254]: Not that I know of, but I can check that out

File: modules/ROOT/pages/clustering/databases.adoc [hunk 12]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+[CAUTION]
+====
+The recreate procedure works only for real user databases and not for composite databases, or the `system` database.
+
+Keep in mind the recreate procedure results in downtime while the stores get updated.
+The time is unbounded and may depend on the store size and network speed.
+
+The recreate procedure selects the most up-to-date seed from the defined seeding servers.
+If only a subset of the previous cluster members is specified as seeders, there is a risk of data loss.
+To avoid this, ensure you back up the current store before replacing it with a seed from a URI, so the latest state can be restored if needed.
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/databases.adoc:248]: This is not really related to the sentence above. There is a risk of data loss when recreating the database if you choose you seeders "wrong" or the system database is not up to date. However, this sentence is more related to the first use case of recreate, when you want to go back in time.

File: modules/ROOT/pages/clustering/databases.adoc (hunk 7 above)
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/databases.adoc:264]:
```suggestion
The store to be used during the recreation of a database can be defined in three different ways.
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 13]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+[[uri-seed]]
+==== `seedURI`
+
+If you provide a URI to a backup or a dump, the stores on all allocations will be replaced by the backup or the dump at the given URI.
+The new allocations can be put on any `ENABLED` server in the cluster.
+See <<cluster-seed-uri, Seed from URI>> for more details.
+
+[source, shell]
+----
+CALL dbms.cluster.recreateDatabase("neo4j", {seedURI: "myBackup"});
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/databases.adoc:279]: I know this was my suggestion in the google doc, but how does this conform with other examples in the documentation? Is it really ok to write "myBackup" for example? Also, another example I saw used the db name "foo", is this also something we should conform to here?

File: modules/ROOT/pages/clustering/databases.adoc (hunk 6 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:257]:
```suggestion
For more information, see xref:clustering/databases.adoc#recreate-databases[Standard databases -> Error handling].
```

File: modules/ROOT/pages/clustering/databases.adoc (hunk 10 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:230]:
```suggestion
== Recreate databases
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1819.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering/databases)
- [Clustering](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering)

File: modules/ROOT/pages/clustering/databases.adoc [hunk 14]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate databases
+
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`] procedure, which allows you:
+
+* To restore a database to a specified backup, while keeping all the associated privileges for the database.
+
+* To restore a database to a write mode again after it has been lost (for example, due to a disaster).
+See xref:clustering/disaster-recovery.adoc[] for more information.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:237]:
```suggestion
// See xref:clustering/disaster-recovery.adoc[] for more information.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1819.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering/databases)
- [Clustering](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1819.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering/databases)
- [Clustering](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering)

File: modules/ROOT/pages/clustering/databases.adoc (hunk 5 above)
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/databases.adoc:253]: Yes, I think here: [CDC](https://development.neo4j.dev/docs/cdc/current/backup-restore/) however, might need some additional info to make it a smooth experience.

Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/databases.adoc:253]: I'll look into adding some more details

File: modules/ROOT/pages/clustering/databases.adoc (hunk 8 above)
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/databases.adoc]: The errors are being put in further up.

File: modules/ROOT/pages/clustering/databases.adoc [hunk 15]
```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+ 
+[CAUTION]
+====
+The recreate procedure works only for real user databases and not for composite databases, or the `system` database.
+
+Keep in mind the recreate procedure results in downtime while the stores get updated.
+The time is unbounded and may depend on the store size and network speed.
+
+The recreate procedure selects the most up-to-date seed from the defined seeding servers.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/databases.adoc]: Do we think this is a bit to advanced for this documentation, and that the spirit of what will happen is still here with this sentence?

[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/databases.adoc]: Natalia will follow up on this.

File: modules/ROOT/pages/clustering/databases.adoc (hunk 13 above)
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/databases.adoc:280]: Made it clearer so should be ok

File: modules/ROOT/pages/clustering/databases.adoc [hunk 16]
```diff
@@ -225,6 +225,110 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+ 
+[CAUTION]
+====
+The recreate procedure works only for real user databases and not for composite databases, or the `system` database.
+
+Keep in mind the recreate procedure results in downtime while the stores get updated.
+The time is unbounded and may depend on different factors -- for example, the size of the store, network speed, etc.
+
+The database recreation process is designed to select the latest seed from the defined seeding servers.
+Specifying only a subset of the previous cluster servers as seeders carries a risk of data loss.
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/databases.adoc:247]: Do we need to think about the dataloss part a bit more.

File: modules/ROOT/pages/clustering/databases.adoc [hunk 17]
```diff
@@ -225,6 +225,124 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+One method uses a backup, while others use available allocations in the cluster.
+
+You can use either <<uri-seed, `seedURI`>> or <<seed-servers, `seedingServers`>> to specify the source from which the database should be recreated.
+
+* If you define neither, an error is thrown.
+
+* If you define both of them, then `seedingServers` must be an empty list.
+
+* If `seedingServers` is not empty and `seedURI` is also defined, an error will occur.
+See <<undefined-servers-backup, Undefined servers with fallback backup>> for more details.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:275]:
```suggestion
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 18]
```diff
@@ -225,6 +225,124 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+=== Seeding options
+
+The store to be used during the recreation of a database can be defined in different ways.
+One method uses a backup, while others use available allocations in the cluster.
+
+You can use either <<uri-seed, `seedURI`>> or <<seed-servers, `seedingServers`>> to specify the source from which the database should be recreated.
+
+* If you define neither, an error is thrown.
+
+* If you define both of them, then `seedingServers` must be an empty list.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:272]:
```suggestion
* If you define both of them, then `seedingServers` must be an empty list.
See <<undefined-servers-backup, Undefined servers with fallback backup>> for more details.
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 19]
```diff
@@ -225,6 +225,124 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+See <<undefined-servers-backup, Undefined servers with fallback backup>> for more details.
+
+[[uri-seed]]
+==== Use backup as a seed
+
+If you provide a URI to a backup or a dump, the stores on all allocations will be replaced by the backup or the dump at the given URI.
+The new allocations can be put on any `ENABLED` server in the cluster.
+See <<cluster-seed-uri, Seed from URI>> for more details.
+
+To avoid possible data loss, ensure you back up the current store before replacing it with a seed from a URI, so the latest state can be restored if needed.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:284]:
```suggestion
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1819.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering/databases)
- [Clustering](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering)

File: modules/ROOT/pages/clustering/databases.adoc [hunk 20]
```diff
@@ -225,6 +225,124 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+[source, shell]
+----
+CALL dbms.cluster.recreateDatabase("neo4j", {seedURI: "s3:/myBucket/myBackup.backup"});
+----
+
+[[seed-servers]]
+==== Use available allocations as seed
+
+The database recreation process is designed to select the latest seed from the seeding servers.
+Specifying only a subset of the previous cluster servers as seeders carries a risk of data loss.
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/databases.adoc:294]: What about this suggestions:

After the recreation has finished, the database will have the most up-to-date store that could be found on the seeding servers. 

And after that in a "caution" block:
Recreation is based on remaining stores or store defined by the user. This means that stores which were lost or not defined is not used for the recreation. If not used stores were more up to date than the used ones, this will result in data loss.

File: modules/ROOT/pages/clustering/databases.adoc [hunk 21]
```diff
@@ -225,6 +225,123 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+The recreate procedure works only for real user databases and not for composite databases, or the `system` database.
+
+Remember that the recreate procedure results in downtime while the stores get updated.
+The time is unbounded and may depend on different factors -- for example, the size of the store, network speed, etc.
+====
+
+The database in question can be in an `online` or `offline` state when it is recreated, but a successful operation starts the database regardless of its previous state.
+
+If CDC is enabled for the database, it will still be enabled after recreating the database, however the chain will be broken.
+See link:https://neo4j.com/docs/cdc/current/backup-restore/[CDC documentation -> Restore backups and snapshots] for more details.
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/databases.adoc:250]: Okay, so I think we should not link to the docs I suggested last time. That handles what the user can see and should do if the database was restored. In the case of recreate, we change database id , so effectively it is a new database. Therefore we should link to how to set CDC up from the beginning. My suggestion is this: 

If CDC is enabled for the database, the chain will stop when the database is recreated. While the recreated database will still have CDC enabled, follow the link:https://development.neo4j.dev/docs/cdc/current/existing-databases/[Initialize CDC applications from an existing database] guide to get it working again.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1819.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering/databases)
- [Clustering](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering)

File: modules/ROOT/pages/clustering/databases.adoc [hunk 22]
```diff
@@ -225,6 +225,129 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+
+[source, shell]
+----
+CALL dbms.cluster.recreateDatabase("neo4j", {seedingServers: ["serverId1", "serverId2", "serverId3"]});
+----
+
+[[undefined-servers]]
+===== Undefined servers
+
+If you provide an empty list of seeding servers and do not specify a `seedURI`, Neo4j automatically selects all available allocations of the database as seeders.
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/databases.adoc:317]: Here we use "Neo4j automatically selects" and down below we use "the system finds" I think we should stick with one of them and use it in both places.

File: modules/ROOT/pages/clustering/databases.adoc [hunk 23]
```diff
@@ -225,6 +225,129 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
...
+
+[source, shell]
+----
+CALL dbms.cluster.recreateDatabase("neo4j", {seedingServers: []});
+----
+
+[[undefined-servers-backup]]
+===== Undefined servers with fallback backup
+
+If both an empty list of seeding servers and a `seedURI` are provided, the system finds all available allocations of the database and use those as seeders.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:328]:
```suggestion
If both an empty list of seeding servers and a `seedURI` are provided, Neo4j finds all available allocations of the database and use those as seeders.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1819.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering/databases)
- [Clustering](https://neo4j-docs-operations-1819.surge.sh/operations-manual/5/clustering)

**Review by AnnaSjerling — APPROVED** [review]
LGTM

**Review by gfx54b — APPROVED** [review]

**Comment by priyolahiri** [comment]
LGTM
